doc/admin/administration.md

```diff
@@ -5,12 +5,137 @@
+
+Queries in-memory health data for the SPDK blobstore (ie NVMe SSD). This
+includes a subset of the SPDK device health stats, as well as I/O error and
+checksum counters.
+```bash
+$ dmg -l boro-11 storage query blobstore-health
+  --devuuid=5bd91603-d3c7-4fb7-9a71-76bc25690c19 -l=boro-11:10001
+boro-11:10001: connected
+Blobstore Health Data:
+boro-11:10001:
+        Device UUID: 5bd91603-d3c7-4fb7-9a71-76bc25690c19
+        Read errors: 0
+        Write errors: 0
+        Unmap errors: 0
+        Checksum errors: 0
+        Device Health:
+                Error log entries: 0
+                Media errors: 0
+                Temperature: 289
+                Temperature: OK
+                Available Spare: OK
+                Device Reliability: OK
+                Read Only: OK
+                Volatile Memory Backup: OK
+```
+
+- Query Persistent Device State: `dmg storage query device-state`
+
+Queries the current persistently stored device state of the specified NVMe SSD
+(either NORMAL or FAULTY).
+```bash
+$ dmg storage query device-state --devuuid=5bd91603-d3c7-4fb7-9a71-76bc25690c19
+-l=boro-11:10001
+boro-11:10001: connected
+Device State Info:
+boro-11:10001:
+        Device UUID: 5bd91603-d3c7-4fb7-9a71-76bc25690c19
+        State: NORMAL
+```
+
+- Manually Set Device State to FAULTY: `dmg storage set nvme-faulty`
+
+Allows the admin to manually set the device state of the given device to FAULTY,
+which will trigger faulty device reaction (all targets on the SSD will be
+rebuilt and the SSD will remain in an OUT state until reintegration is
+supported).
+```bash
+$ dmg storage set nvme-faulty --devuuid=5bd91603-d3c7-4fb7-9a71-76bc25690c19
+-l=boro-11:10001
+boro-11:10001: connected
+Device State Info:
+boro-11:10001:
+        Device UUID: 5bd91603-d3c7-4fb7-9a71-76bc25690c19
+        State: FAULTY
+```
+
 ## System Operations
 
 ### Full Shutdown and Restart
 
-Details on how to support proper DAOS server shutdown will be provided
-in a future revision.
+A DAOS system can be restarted after a controlled shutdown providing
+no configurations changes have been made after initial format.
+
+The DAOS Control Server instance acting as access point records DAOS
+I/O Server instances that join the system in a "membership".
+
+When up and running, the entire system (all I/O Server instances)
+can be shutdown with the command
+`dmg -l <access_point_addr> system stop`, after which DAOS Control
+Servers will continue to operate and listen on the management network.
+
+To start the system again (with no configuration changes) after a
+controlled shutdown, run the command
+`dmg -l <access_point_addr> system start`, DAOS I/O Servers
+managed by DAOS Control Servers will be started.
+
+To query the system membership, run the command
+`dmg -l <access_point_addr> system query`, this lists details
+(rank/uuid/control address/state) of DAOS I/O Servers in the
+system membership.
+
+!!! warning
```

Comment:
Doesn't seem like this is rendered using github markdown. At least, it doesn't seem to do anything in the rich diff view.

Reply:
Oh, i see. This is for mkdocs.